File: AlbumLibrary/src/main/java/com/album/ui/widget/SimpleAlbumListener.java
package com.album.ui.widget;

import android.support.annotation.NonNull;
import android.support.annotation.Nullable;

import com.album.AlbumListener;
import com.album.model.AlbumModel;

import java.io.File;
import java.util.List;

/**
 * by y on 18/08/2017.
 */

public class SimpleAlbumListener implements AlbumListener {

    @Override
    public void onAlbumActivityFinish() {

    }

    @Override
    public void onAlbumPermissionsDenied(int type) {

    }

    @Override
    public void onAlbumFragmentNull() {

    }

    @Override
    public void onAlbumPreviewFileNull() {

    }

    @Override
    public void onAlbumFinderNull() {

    }

    @Override
    public void onAlbumBottomPreviewNull() {

    }

    @Override
    public void onAlbumBottomSelectNull() {

    }

    @Override
    public void onAlbumFragmentFileNull() {

    }

    @Override
    public void onAlbumPreviewSelectNull() {

    }

    @Override
    public void onAlbumCheckBoxFileNull() {

    }

    @Override
    public void onAlbumFragmentCropCanceled() {

    }

    @Override
    public void onAlbumFragmentCameraCanceled() {

    }

    @Override
    public void onAlbumFragmentUCropError(@Nullable Throwable data) {

    }

    @Override
    public void onAlbumResources(@NonNull List<AlbumModel> list) {

    }

    @Override
    public void onAlbumUCropResources(@Nullable File scannerFile) {

    }

    @Override
    public void onAlbumMaxCount() {

    }

    @Override
    public void onAlbumActivityBackPressed() {

    }

    @Override
    public void onAlbumOpenCameraError() {

    }

    @Override
    public void onAlbumEmpty() {

    }

    @Override
    public void onAlbumNoMore() {

    }

    @Override
    public void onAlbumResultCameraError() {

    }
}

File: AlbumLibrary/src/main/java/com/album/ui/widget/SimpleAlbumListener.kt
package com.album.ui.widget

import com.album.entity.AlbumEntity
import com.album.listener.AlbumListener
import java.io.File

/**
 * by y on 18/08/2017.
 */

open class SimpleAlbumListener : AlbumListener {

    override fun onAlbumResultCameraError() {
    }

    override fun onAlbumActivityFinish() {

    }

    override fun onAlbumPermissionsDenied(type: Int) {

    }

    override fun onAlbumFragmentNull() {

    }

    override fun onAlbumPreviewFileNull() {

    }

    override fun onAlbumFinderNull() {

    }

    override fun onAlbumBottomPreviewNull() {

    }

    override fun onAlbumBottomSelectNull() {

    }

    override fun onAlbumFragmentFileNull() {

    }

    override fun onAlbumPreviewSelectNull() {

    }

    override fun onAlbumCheckBoxFileNull() {

    }

    override fun onAlbumFragmentCropCanceled() {

    }

    override fun onAlbumFragmentCameraCanceled() {

    }

    override fun onAlbumFragmentUCropError(data: Throwable?) {

    }

    override fun onAlbumResources(list: List<AlbumEntity>) {

    }

    override fun onAlbumUCropResources(scannerFile: File) {

    }

    override fun onAlbumMaxCount() {

    }

    override fun onAlbumActivityBackPressed() {

    }

    override fun onAlbumOpenCameraError() {

    }

    override fun onAlbumEmpty() {

    }

    override fun onAlbumNoMore() {

    }

    override fun onVideoPlayError() {
    }
}
